applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -1,4 +1,5 @@
 import KratosMultiphysics
+from CompressiblePotentialFlowApplication import*
```

Comment:
better to use `import KratosMultiphysics.CompressiblePotentialFlowApplication as CompressiblePotentialFlowApplication` or so

Reply:
why?

Reply:
It is better to know where things come from

imagine two apps define the same fct, which one is used?
Helps a lot with debugging
Also this is what is used in Kratos in up-to-date code

imo this is similar to not doing `using namespace std` in C

Reply:
Agreed, but then each time I want to use a function one needs to write the whole CompressiblePotentialFlowApplication.function, isn't that adding to much text and making the code not so clean?

Reply:
Some developers short it like

`import ... as KratosCPS`

Reply:
yep also wanted to suggest that

Reply:
Btw clean code does imo not refer to how long/short the names are

It should be understandable, and using the full name definitely helps with that

Reply:
Ok, thank you both for the hints! I will definitely take them into account for the future.